modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -281,6 +282,9 @@ if (CONFIG_BUILD_WITH_TFM)
       -DTFM_TOOLCHAIN_FILE=${ZEPHYR_TRUSTED_FIRMWARE_M_MODULE_DIR}/${TFM_TOOLCHAIN_FILE}
       -DCROSS_COMPILE=${TFM_TOOLCHAIN_PATH}/${TFM_TOOLCHAIN_PREFIX}
       -DCMAKE_BUILD_TYPE=${TFM_CMAKE_BUILD_TYPE}
+      # Force DRARF version 4 due to lack of support for version 5 in
+      # pyelftools, remove once support is added
+      -DCMAKE_C_FLAGS_${TFM_CMAKE_BUILD_TYPE_UPPER}=${CMAKE_C_FLAGS_${TFM_CMAKE_BUILD_TYPE_UPPER}}\ -gdwarf-4
```

Comment:
I suppose this will conflict with the -g from CMAKE_BUILD_TYPE.

Perhaps upstream TF-M needs to be modified to use dwarf-4 (either conditionally or unconditionally).